app/views/dashboard/projects/index.html.erb

```diff
@@ -48,30 +38,25 @@
             end
           ),
         aria: {
-          current: params[:personal] ? "page" : "false"
+          current: params[:personal] ? "page" : "false",
         } %>
       </li>
     </ul>
     <div class="flex flex-row items-center ml-auto space-x-2 font-normal">
       <%= search_form_for @q, url: dashboard_projects_url(**request.query_parameters.slice(:personal)), html: {"data-controller": "filters"} do |f| %>
```

Comment:
I'm assuming the `.slice(:personal)` was trying to pass the personal param and can now be removed since we've added the personal param as an input

Reply:
Removed in 8a925e9